### DISCUSSION TITLE
Taskbar navigation

### DISCUSSION THREAD

**Discussion by Dara1LT** [pear-devs/pear-desktop#1869] [Ideas]
title, would be great to have taskbar navigation, |<[play]>|

**Comment by JellyBrick** [comment]
![image](https://github.com/th-ch/youtube-music/assets/16558115/64fa8ecf-b707-4e2c-ba74-c2d5784cafa7)

**Reply by Dara1LT** [reply]
![image](https://github.com/th-ch/youtube-music/assets/113183537/65ff3c33-eeb7-4a46-950a-152551dea059)
sadly no such thing on w11, unless its a plugin (it was a plugin never mind that)
![image psd](https://github.com/th-ch/youtube-music/assets/113183537/100369a8-41ef-4800-8526-1e9083f2281d)
I was thinking something like that
